pkg/runs/submitter.go

```diff
@@ -88,6 +88,8 @@ func (submitter *Submitter) executePortfolio(portfolio *Portfolio,
 	// Build list of runs to submit
 	readyRuns := submitter.buildListOfRunsToSubmit(portfolio, runOverrides)
 
+	log.Printf("executePortfolio - readyRuns %v", readyRuns)
```

Comment:
This will cause a ton of things to be logged. Do we still need this one here ?

Reply:
will remove!